spring-batch-docs/asciidoc/step.adoc

```diff
@@ -1394,10 +1401,12 @@ public class SkipCheckingListener extends StepExecutionListenerSupport {
 
 The above code is a `StepExecutionListener`
         that first checks to make sure the `Step` was
-        successful, and next if the skip count on the
+        successful and next if the skip count on the
         `StepExecution` is higher than 0. If both
         conditions are met, a new `ExitStatus` with an
-        exit code of "COMPLETED WITH SKIPS" is returned.
+        exit code of `COMPLETED WITH SKIPS` is returned.
+// TODO "and next if the skip count on the `StepExecution` is higher than 0" confuses me.
+// Please explain that, and I'll make a stronger sentence here.
```

Comment:
Spring batch keeps track of the items that are skipped during processing.  The listener above checks if the step is successful (`if(!exitCode.equals(ExitStatus.FAILED.getExitCode())`) then it checks if the number of items skipped is more than 0 (`stepExecutino.getSkipCount() > 0`).